src/features/events/components/EventPopper/SingleEvent.tsx

```diff
@@ -272,38 +294,57 @@ const SingleEvent: FC<SingleEventProps> = ({ event, onClickAway }) => {
           </Box>
         </Box>
       )}
-      <Box
-        alignItems="center"
-        display="flex"
-        justifyContent="flex-end"
-        marginBottom={2}
-      >
-        <NextLink href={getEventUrl(event)} legacyBehavior passHref>
-          <Link underline="none">
+      {showPublishButton ? (
```

Comment:
I think this ternary should be replaced with two separate expressions on the pattern `{showPublishButton && (<></>)}` instead. In a long vertical chunk of code, I find it very difficult to find the place where the ternary "switches" from the expr-if-true to expr-if-false.